obspy/io/hypodd/pha.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+"""
+HypoDD PHA read support.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+import io
+
+from obspy import UTCDateTime
+from obspy.core.event import (
+    Catalog, Event, Origin, Magnitude, Pick, WaveformStreamID, Arrival,
+    OriginQuality)
+
+
+def _seed_id_map(inventory=None, id_map=None, id_default='.{}..{}'):
+    if id_map is None:
+        id_map = {}
+    ret = id_map.copy()
+    if inventory is not None:
+        for net in inventory:
+            for sta in net:
+                if len(sta) == 0:
+                    temp = id_map.get(sta.code, id_default)
+                    temp = temp.split('.', 2)[-1]
```

Comment:
I was not aware of this extra argument to the `.split()` method. Very cool! :-) Would be a bit more readable if you use `maxsplit=2`.

Reply:
I use it all the time.. :-)

Reply:
Btw.. `split()` doesn't even work with named kwargs, I guess it's some Python C API thingy? I could never be bothered to look up the reason..

Reply:
A side effect is that you can't use `maxsplit` without specifying the separator, even if you want the default one, so you have to say `split(None, 2)` in that case. Weird non-Pythonic glitch somehow.. ;-)

Reply:
The non-named kwarg thing is Python 2 only, I think.

Reply:
Ah OK. I'll probably never tried it on Py3 ;-)